extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_linkAuthentication.php

```diff
@@ -1,10 +1,27 @@
 <?php if ( isset( $model['subtitle'] ) ): ?>
-	<div class="wds-global-navigation__account-menu-dropdown-caption"><?= DesignSystemHelper::renderText( $model['subtitle'] ) ?></div>
+	<div class="wds-global-navigation__account-menu-dropdown-caption">
+		<?= DesignSystemHelper::renderText( $model['subtitle'] ) ?>
+	</div>
+<?php endif; ?>
+
+<?php if ( $model['title']['key'] === 'global-navigation-user-sign-out'): ?>
+	<div class="<?= $classNames ?> wds-sign-out"
+	     id="<?= Sanitizer::encodeAttribute( $model['title']['key'] ); ?>">
```

Comment:
this data comes from external source so it's not fully safe. Additionally this API doesn't ensure this data is HTML safe - at some time it may return `"` there.